### ISSUE TITLE
Compilation Error

### ISSUE THREAD

**Issue by umerhuzaifa** [flexible-collision-library/fcl#124]
Hi,

I am facing the following error when compiling a modified version of "general_test.cpp" available in the fcl-master/test folder. As I run "catkin_make" in my catkin workspace,  this is the error I get:

`[ 80%] Built target tracking_node
CMakeFiles/planner3.dir/src/planner3.cpp.o: In function `Box':
/opt/ros/indigo/include/fcl/collision_object.h:63: undefined reference to `fcl::AABB::AABB()'
CMakeFiles/planner3.dir/src/planner3.cpp.o: In function `Box':
/opt/ros/indigo/include/fcl/shape/geometric_shapes.h:78: undefined reference to `vtable for fcl::Box'
CMakeFiles/planner3.dir/src/planner3.cpp.o: In function `Box':
/opt/ros/indigo/include/fcl/collision_object.h:63: undefined reference to `fcl::AABB::AABB()'
CMakeFiles/planner3.dir/src/planner3.cpp.o: In function `Box':
/opt/ros/indigo/include/fcl/shape/geometric_shapes.h:78: undefined reference to `vtable for fcl::Box'
collect2: error: ld returned 1 exit status
make[2]: *** [---/catkin_ws/devel/lib/assign3/planner3] Error 1
make[1]: *** [---/CMakeFiles/planner3.dir/all] Error 2
make: *** [all] Error 2
`
Kindly guide what can be done to fix these errors?

Thanks,
Umer

**Comment by scpeters** [comment]
Since you're compiling the ROS version of this package, can you also post this as a question on https://answers.ros.org ? If it's a problem with `catkin_make`, they will be more likely to know the answer.

**Comment by jslee02** [comment]
I agree with @scpeters . It seems like the consumer project's (e.g., `planner3`) problem that isn't correctly linking to FCL rather than FCL itself. Any luck on http://answers.ros.org/questions/?
